CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the maintainers of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
```

Comment:
We aren't using react here so there won't any build dependencies, remove this line

Reply:
done